packages/core/src/reporters/ProgressKeeper.ts

```diff
@@ -32,32 +32,36 @@ abstract class ProgressKeeper implements Reporter {
     }
   }
 
+  protected getElapsedTime() {
+    const elapsedSeconds = this.timer.elapsedSeconds();
+
+    return this.formatTime(
+      // hours:
+      Math.floor(elapsedSeconds / 3600),
+      // minutes up to 2 decimal places:
+      Math.floor(((elapsedSeconds % 3600) * 100) / 60) / 100
+    );
+  }
+
   protected getEtc() {
     const totalSecondsLeft = Math.floor((this.timer.elapsedSeconds() / this.progress.tested) * (this.progress.total - this.progress.tested));
 
     if (isFinite(totalSecondsLeft) && totalSecondsLeft > 0) {
-      const hours = Math.floor(totalSecondsLeft / 3600);
-      const minutes = Math.floor((totalSecondsLeft / 60) % 60);
-      const seconds = Math.floor(totalSecondsLeft % 60);
-
-      return this.formatEtc(hours, minutes, seconds);
+      return this.formatTime(
+        // minutes up to 2 decimal places:
+        Math.floor(totalSecondsLeft / 3600),
+        // minutes up to 2 decimal places:
+        Math.floor(((totalSecondsLeft % 3600) * 100) / 60) / 100
+      );
     } else {
       return 'n/a';
     }
   }
 
-  private formatEtc(hours: number, minutes: number, seconds: number) {
-    let output;
-
-    if (hours > 0) {
-      output = `${hours}h, ${minutes}m, ${seconds}s`;
-    } else if (minutes > 0) {
-      output = `${minutes}m, ${seconds}s`;
-    } else {
-      output = `${seconds}s`;
-    }
-
-    return output;
+  private formatTime(hours: number, minutes: number) {
+    return hours > 0 // conditional time formatting
+      ? `${hours}h, ${minutes}m`
```

Comment:
Another possible, minor improvement to hide the decimal places after 1 hour:

```suggestion
      ? `${hours}h, ${Math.floor(minutes)}m`
```

This would need a test update.